html.md

```diff
@@ -0,0 +1,32 @@
+# HTML
+
+## Table of Contents
+
+1. [Tables](#tables)
+
+## Tables
+
+Use the `scope` attribute to identify if a table header is a row or column header. Always use a descriptive `caption` and "visually hide" it if necessary for the design.
```

Comment:
Austin just mentioned this IRL, so I'll suggest wording to come at the start or end of this:

> HTML tables should be exclusively used for tabular data, such as usage reports or structured information (see example). They should not be used for layout. The only possible exception being email templates when necessary.

The line about email templates could be left off if it's distracting.